components/support/nimbus-fml/src/cli.yaml

```diff
@@ -27,15 +27,18 @@ args:
         long: output
         value_name: FILE
         global: true
+    - channel:
```

Comment:
Happy to take suggestions on the best place to put this argument! I put it here because it seem like it's not super specific to android (i.e ios has beta and release) and I needed the argument when I was running the `intermediate-repr` command to validate that all is good